CHANGELOG.md

```diff
@@ -28,6 +28,23 @@
 ### Jsonnet
 
 * [BUGFIX] Fixed query-scheduler ring configuration for dedicated ruler's queries and query-frontends. #3237 #3239
+* [CHANGE] [Common storage configuration](https://grafana.com/docs/mimir/v2.3.x/operators-guide/configure/configure-object-storage-backend/#common-configuration) is now used to configure object storage in all components. This is a breaking change in terms of Jsonnet manifests and also a CLI flag update for components that use object storage, so it will require a rollout of those components. The changes include: #3257
```

Comment:
[nit] Please move it above the `[BUGFIX]`

Reply:
Done in https://github.com/grafana/mimir/pull/3257/commits/6ec7971b264c6f7a0682a36c7749ae9ce29b0444